blockchain/state_migration.go

```diff
@@ -376,8 +375,8 @@ func (bc *BlockChain) StopStateMigration() error {
 
 // StateMigrationStatus returns if it is in migration, the block number of in migration,
 // number of committed blocks and number of pending blocks
-func (bc *BlockChain) StateMigrationStatus() (bool, uint64, int, int, float64, error) {
-	return bc.db.InMigration(), bc.db.MigrationBlockNumber(), bc.committedCnt, bc.pendingCnt, bc.progress, bc.migrationErr
+func (bc *BlockChain) StateMigrationStatus() (bool, uint64, int, int, int, float64, error) {
```

Comment:
There are too many return variables. How about declaring a struct for state migration status?

Reply:
How about this function returns `map[string]interface{}`?
If it returns a struct, we need to declare the struct type.
But I can't find good place where I define the struct type.
Any good place?

Reply:
If API is the only place where `StateMigrationStatus` is used, I'm okay with returning the result as as a map.